app/grandchallenge/reader_studies/serializers.py

```diff
@@ -109,20 +125,29 @@ def validate(self, attrs):
                 raise ValidationError(
                     "This reader study does not allow answer modification."
                 )
-            if list(attrs.keys()) != ["answer"]:
+            if (
+                list(attrs.keys()) != ["answer"]
+                or "images" in self.initial_data
+            ):
                 raise ValidationError("Only the answer field can be modified.")
             question = self.instance.question
-            images = self.instance.images.all()
+            civs = self.instance.civs.all()
             creator = self.instance.creator
         else:
             question = attrs.get("question")
-            images = attrs.get("images")
+            civs = ComponentInterfaceValue.objects.filter(
```

Comment:
Could there be a problem here if there are multiple CIVs for an image? Maybe we need to filter by reader study?

Reply:
Ah yes, possibly. Nice catch.